components/gitpod-protocol/src/util/tracing.ts

```diff
@@ -39,7 +49,22 @@ export namespace TraceContext {
             "error": err.message,
             "stacktrace": err.stack
         })
-        ctx.span.setTag("error", true)
+        ctx.span.setTag("error", true);
+        if (errorCode) {
+            ctx.span.setTag("errorCode", errorCode);
```

Comment:
I know we haven't started documented our naming conventions for tags (attributes OpenTelemetry-land) but if this is a HTTP status I'd prefer `http.status_code` as the tag name. There is a bit of [docs here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md) but we will eventually write our own ([see issue](https://github.com/gitpod-io/ops/issues/332))

Reply:
If this is just a generic errorCode then sticking with errorCode for now is fine

Reply:
TIL that there are conventions for this! :bulb: 

We use jsonrpc, so this fits: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/rpc.md#json-rpc